guides/developer/plugin-creators/overview.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: single
+title: "Overview"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+<div class="notice--danger">
+  <strong>Note:</strong> Plugins are an alpha feature that is under active development and may change.
+</div>
+
+
+# Requirements
+
+* Spinnaker v1.19.0
+* Orca branch `release-1.19.x` (for local testing)
+* IntelliJ IDEA (for local testing)
+
+# Plugin overview
```

Comment:
We need a taxonomy section. This PR is mixing terminology up and it'll confuse people.

* Extension Point: Defined by a Spinnaker service, it represents an official contract for which extensions can be written against. Written by Core Contributors.
* Extension: A piece of code that implements an Extension Point. Written by Developers.
* Plugin: A collection of Extensions that can be bundled together and shipped as a single artfiact. Also written by Developers.

Reply:
added